mlflow/store/artifact/utils/models.py

```diff
@@ -16,8 +16,8 @@ def is_using_databricks_registry(uri):
 def _improper_model_uri_msg(uri):
     return (
         "Not a proper models:/ URI: %s. " % uri
-        + "Models URIs must be of the form 'models:/<model_name>/<suffix>' "
```

Comment:
This error message doesn't render well in notebooks, since `<model_name>` is rendered/escaped as an HTML tag; fix it by removing the angle brackets

Reply:
Didn't know that, makes sense!

Reply:
<img width="697" alt="image" src="https://github.com/mlflow/mlflow/assets/17039389/11b0d1a5-6c36-406a-a82d-483e772d07b0">

lol

Reply:
We should probably look for other cases of this.... I had no idea that it would parse as a tag

Reply:
<img width="697" alt="image" src="https://github.com/mlflow/mlflow/assets/17039389/1ed234b1-1631-4ec9-92ba-c4d30ef2cae7">